src/main/java/de/dhbw/karlsruhe/model/Difficulty.java

```diff
@@ -0,0 +1,29 @@
+package de.dhbw.karlsruhe.model;
+
+import java.util.stream.Stream;
+
+public enum Difficulty {
+    EASY("e"),
+    MEDIUM("m"),
+    HARD("h");
+
+    public final String shortDifficultyName;
+
+    Difficulty(String shortDifficultyName) {
+        this.shortDifficultyName = shortDifficultyName;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Musst du die toString Methode hier überschreiben? Könntest du beim Iterrieren nicht einfach object.name() aufrufen?

Reply:
Ich habe mal die toString Methode genutzt, falls wir die Schwierigkeitsnamen anders ausgeben wollen, zum Beispiel alles klein  oder ähnliches. Könnte das natürlich anpassen, bin mir nicht sicher, was besser ist. Was denkst du?

Reply:
Alles klar. Dann können wir das so lassen. Denkst du, der Name ist so weit auch ok, oder sollten wir die Methode umbenennen bspw. in getName()? Denke, beides wäre in Ordnung.

Reply:
Habe ich so angepasst, da das momentan am eindeutigsten ist.